python/py-pymysql/Portfile

```diff
@@ -14,15 +14,15 @@ maintainers         {@xeron gmail.com:xeron.oskom} openmaintainer
 description         Pure-Python MySQL client library
 long_description    ${description}
 
-python.versions     27 36 37 38
+python.versions     27 36 37 38 39
```

Comment:
version 1.0.2 does not support PY27 anymore, so you'll need to restrict it to the latest supported version for the py27 subport